functions/index.js

```diff
@@ -41,3 +43,47 @@ exports.dailycc = functions.https.onRequest(() => {
 		}
 	};
 });
+
+exports.affirmations = functions.https.onRequest((req, res) => {
+	const client = new Discord.Client();
+	const { access_token } = functions.config().discord;
+
+	client.login(access_token).catch((err) => console.log(err));
+
+	client.on("ready", async () => {
+		postAffirmation(client.channels.cache.get("771821709589479505"));
+	});
+
+	const postAffirmation = async (channel) => {
+		const result = await getAffirmation();
+
+		if (result === null || result === undefined) {
+			channel.send(`Cheers to a new week 😁`);
+			return;
+		} else {
+			channel.send(result).catch((err) => {
+				channel.send("There was an issue grabbing the affirmation");
+				channel.send(`Error: ${err.message}`);
+			});
+		}
+	};
+
+	const getAffirmation = async () => {
+		try {
+			const { affirmation } = await admin
+				.firestore()
+				.collection("constants")
+				.doc("affirmations")
+				.get()
+				.then((res) => {
+					console.log(res.data());
+					return res;
+				})
+				.catch((err) => res.send(err));
+			res.send(affirmation);
```

Comment:
We don't need to send anything back. Which is why I made the comment above about not needing req and res.

Reply:
See the previous comment, we need to make sure we send a response. This is good.